devicon.json

```diff
@@ -2158,6 +2158,24 @@
         "color": "#000000",
         "aliases": []
     },
+    {
```

Comment:
altnames needs to be added, see [updating the devicon.json](https://github.com/devicons/devicon/wiki/Updating-the-%60devicon.json%60)